fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/RegrAvgXToAvg.java

```diff
@@ -0,0 +1,44 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.rules.expression.rules;
+
+import org.apache.doris.nereids.rules.expression.ExpressionPatternMatcher;
+import org.apache.doris.nereids.rules.expression.ExpressionPatternRuleFactory;
+import org.apache.doris.nereids.trees.expressions.Expression;
+import org.apache.doris.nereids.trees.expressions.functions.agg.Avg;
+import org.apache.doris.nereids.trees.expressions.functions.agg.RegrAvgX;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * RegrAvgXToAvg
+ */
+public class RegrAvgXToAvg implements ExpressionPatternRuleFactory {
```

Comment:
regr_avgx and regr_avgy is not same with avg, we could not rewrite them to avg. refer to  https://docs.snowflake.com/en/sql-reference/functions/regr_avgx

Reply:
ok